README.md

```diff
@@ -67,17 +75,17 @@ This route allows your GitHub Actions job to post a message in a Slack channel o
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
-## Slack Incoming Webhook Route
+## Technique 3: Slack Incoming Webhook
 
-This route allows your GitHub Actions job to post a message in a Slack channel or direct message by utilizing [Incoming Webhooks](https://api.slack.com/messaging/webhooks). With this route, you can instantly post a message.
+This approach allows your GitHub Actions job to post a message to a Slack channel or direct message by utilizing [Incoming Webhooks](https://api.slack.com/messaging/webhooks).
 
-Incoming Webhooks conform to the same rules and functionality as any of our other messaging APIs. You can make your posted messages as simple as a single line of text, or make them really useful with [interactive components](https://api.slack.com/messaging/interactivity). To make the message more expressive and useful use [Block Kit](https://api.slack.com/block-kit) to build and test visual components.
+Incoming Webhooks conform to the same rules and functionality as any of Slack's other messaging APIs. You can make your posted messages as simple as a single line of text, or make them really useful with [interactive components](https://api.slack.com/messaging/interactivity). To make the message more expressive and useful use [Block Kit](https://api.slack.com/block-kit) to build and test visual components.
 
 ## Setup
 
-* [Create a Slack App](https://api.slack.com/apps) for your workspace (alternatively use an existing app you have already created and installed)
-* Add the [incoming-webhook](https://api.slack.com/scopes/incoming-webhook) bot scope under **OAuth & Permissions**
-* Install the app to your workspace (you will select a channel to notify)
+* [Create a Slack App][apps] for your workspace (alternatively use an existing app you have already created and installed).
+* Add the [`incoming-webhook`](https://api.slack.com/scopes/incoming-webhook) bot scope under **OAuth & Permissions**.
+* Install the app to your workspace (you will select a channel to notify).
```

Comment:
I think we should add a step on how / where to get and set the `SLACK_WEBHOOK_URL` env var in this case. Any tips?

Reply:
Yeah, not a bad idea. You get it from the app config page under the *Incoming Webhooks* section

Reply:
Done!